src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -496,19 +496,21 @@ public void testPanelsCollapseExpand() {
         assertEquals("Collapse Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Collapse all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
-        assertTrue(resultsPage.verifyAllResultsPanelBodyVisibility(true));
+        assertTrue(resultsPage.areResultsVisible());
 
         resultsPage.clickCollapseExpand();
+        resultsPage.waitForPanelsToCollapse();
         assertEquals("Expand Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Expand all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
-        assertTrue(resultsPage.verifyAllResultsPanelBodyVisibility(false));
+        assertTrue(resultsPage.areResultsHidden());
 
         resultsPage.clickCollapseExpand();
+        resultsPage.waitForPanelsToExpand();
         assertEquals("Collapse Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Collapse all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
-        assertTrue(resultsPage.verifyAllResultsPanelBodyVisibility(true));
+        assertTrue(resultsPage.areResultsVisible());
```

Comment:
If all the methods are wrapped with `assertTrue(...)`, consider renaming the method to `verifyResultsVisible/Hidden` instead and move the assertions to the respective methods.